tests/erp/test_browser.py

```diff
@@ -483,6 +483,7 @@ def mock_geocode(*args, **kwargs):
         data={
             "accueil_visibilite": True,
             "accueil_personnels": "aucun",
+            "acceuil_equipements_malentendants_presence": False,
             "accueil_equipements_malentendants": ["bim", "lsf"],
```

Comment:
Logically, `accueil_equipements_malentendants_presence` should  be `True` if we actually select equipments :)

Reply:
Yeah, i was waiting for feedback before re-running the tests thanks :)